src/python/doc/representations.rst

```diff
@@ -80,3 +133,11 @@ Metrics
    :members:
    :special-members:
    :show-inheritance:
+
+PersLay
+-------
+.. automodule:: gudhi.tensorflow.perslay
```

Comment:
We may want to specify the order in which the classes appear in the doc. By default, this is the lexicographic order. But it could make sense to group the Phi together, group the Weight separately, and have the main Perslay at one extremity.